### DISCUSSION TITLE
Adding another language for Belgium

### DISCUSSION THREAD

**Discussion by anfourny** [FossifyOrg/General-Discussion#422] [General]
The Belgian holidays are currently only written in Dutch. I would like the option for a French or English version of holidays in Belgium too. I am happy to contribute translations for these myself, but I don't know how to do it exactly. Is there a way to update the .ics file for Belgium so that it can manage several languages? Or should it be a new option, such as "Belgium-fr" or "Belgium-en"?

**Comment by Aga-C** [comment]
We don't plan on providing such an option. We'd need to add translations of all holidays for all countries, what would be too much work. More sense would be to add names for each official language (AFAIK English is not an official language of Belgium), but it's up to creators of each ICS file to do them as universal as possible.
